tawazi/helpers.py

```diff
@@ -66,6 +67,15 @@ def get_args_and_default_args(func: Callable[..., Any]) -> Tuple[List[str], Dict
     return args, default_args
 
 
+def make_raise_arg_error(func_name: str, arg_name: str) -> Callable[[], None]:
```

Comment:
this name is a bit weird imo, could we make that a bit clearer ?
And why don't you just return `lambda fname, argname : raise_arg_exc(fname, argname)` ?

Reply:
Done